src/main/java/org/dbflute/intro/app/web/dfprop/DfpropAction.java

```diff
@@ -155,12 +164,34 @@ private SchemaPolicyMap mappingToSchemaPolicyMap(DfpropEditSchemaPolicyBody body
         String builtStatement = dfpropUpdateLogic.registerSchemaPolicyStatement(clientName, statement);
         return asJson(builtStatement);
     }
+
     private SchemaPolicyStatement mappingToStatement(DfpropRegisterSchemaPolicyStatementBody body) {
         SchemaPolicyStatement.Condition condition = new SchemaPolicyStatement.Condition(body.condition.operator, body.condition.conditions);
         SchemaPolicyStatement.Expected expected = new SchemaPolicyStatement.Expected(body.expected.operator, body.expected.expected);
         return new SchemaPolicyStatement(body.type, body.subject, condition, expected, body.comment);
     }
 
+    // -----------------------------------------------------
+    //                       GetschemapolicyStatementSubject
+    //                       -------------------------------
+    @Execute(urlPattern = "{}/@word/@word/@word")
+    public JsonResponse<List<String>> schemapolicyStatementSubject(String clientName) {
+        ArrayList<String> tablemapSubjectList = new ArrayList<String>();
```

Comment:
At least, extract it to logic class.

Reply:
@jflute 
fixed.
fc7836ae7dab037d8de2f2e044e1fb5ae6129bf1
8e5c1e3bb67a5e8e961a1d498a118ec8ac78624c